packages/shared/src/utils/proxy.test.ts

```diff
@@ -24,6 +24,24 @@ describe('isProxyUrlRelative(key)', () => {
   });
 });
 
+describe('isHttpOrHttps(key)', () => {
+  it('returns true if the proxyUrl starts with `http://`', () => {
```

Comment:
🙃 Check also it.each. 

Please add a test case for empty string.

Reply:
thank you for the suggestions. I added them